genienlp/predict.py

```diff
@@ -445,6 +445,16 @@ def parse_argv(parser):
         help='if true the provided dataset would not contain the answer (translated sentence)',
     )
     parser.add_argument('--plot_heatmaps', action='store_true', help='whether to plot cross-attention heatmaps')
+    parser.add_argument(
+        '--replace_qp',
```

Comment:
This flag is not very understandable, can we choose a better name?
Also, rather than have two flags, maybe have one flag with parameter `yes`, `no`, `force`?

Reply:
I'm actually filing a PR after this which changes the way we do translation + alignment. I'm removing the `force_replace_qp` there. As that PR is backward incompatible by nature, I would do the renaming there and update all the k8s pipelines.